main.py

```diff
@@ -23,7 +47,8 @@ def new_start(*args, **kwargs):
     def preexec_function():
         os.setpgrp()
     default_Popen = subprocess.Popen
-    subprocess.Popen = functools.partial(subprocess.Popen, preexec_fn=preexec_function)
+    subprocess.Popen = functools.partial(
+        subprocess.Popen, preexec_fn=preexec_function)
```

Comment:
Could you revert these kind of style changes in order to keep the diff clean? There's a few throughout this PR. I will add black when I get half a chance, to solve this kind of thing properly :)

Reply:
fixed